lms/tasks/roster.py

```diff
@@ -4,22 +4,37 @@
 
 from sqlalchemy import exists, select
 
-from lms.models import Course, CourseRoster, Event, LMSCourse
+from lms.models import (
+    Assignment,
+    AssignmentRoster,
+    Course,
+    CourseRoster,
+    Event,
+    LMSCourse,
+)
 from lms.services.roster import RosterService
 from lms.tasks.celery import app
 
-COURSE_LAUNCHED_WINDOW = timedelta(hours=24)
-"""How recent we need to have seen a launch from a course before we stop fetching rosters for it."""
+LAUNCHED_WINDOW = timedelta(hours=24)
+"""How recent we need to have seen a launch from a course/assignment before we stop fetching rosters for it."""
 
 ROSTER_REFRESH_WINDOW = timedelta(hours=24 * 7)
-"""How frequenly should we fetch roster for the same course"""
+"""How frequenly should we fetch roster for the same course/assignment"""
 
-ROSTER_COURSE_LIMIT = 5
+ROSTER_LIMIT = 5
 """How many roster should we fetch per executing of the schedule task."""
 
 
 @app.task()
 def schedule_fetching_rosters() -> None:
+    # Schedule fetching course rosters
+    schedule_fetching_course_rosters()
+
+    # Schedule fetching assignment rosters
+    schedule_fetching_assignment_rosters()
```

Comment:
The two comments seem a bit unnecessary, as they say basically the same as the function names.

```suggestion
    schedule_fetching_course_rosters()
    schedule_fetching_assignment_rosters()
```